ConfigurationProfileSettings/Readme.md

```diff
@@ -58,8 +58,11 @@ Microsoft Edge\Sites that can access video capture devices without requesting pe
+Microsoft Edge\Startup, home page and new tab page\Configure the new tab page URL | Configures the default URL for the new tab page. This policy determines the page that's opened when new tabs are created (including when new windows are opened). It also affects the startup page if that's set to open to the new tab page. This policy doesn't determine which page opens on startup; that's controlled by the 'RestoreOnStartup' (Action to take on startup) policy. It also doesn’t affect the home page if that’s set to open to the new tab page. If you don't configure this policy, the default new tab page is used. If you configure this policy *and* the 'NewTabPageSetFeedType' (Configure the Microsoft Edge new tab page experience) policy, this policy has precedence. If an invalid URL is provided, new tabs will open about://blank. This policy is available only on Windows instances that are joined to a Microsoft Active Directory domain or Windows 10 Pro or Enterprise instances that are enrolled for device management. Example value: https://www.fabrikam.com | https://www.bing.com
+Microsoft Edge\Startup, home page and new tab page\Set the new tab page as the home page | Configures the default home page in Microsoft Edge. You can set the home page to a URL you specify or to the new tab page. If you enable this policy, the new tab page is always used for the home page, and the home page URL location is ignored. If you disable this policy, the user's home page can't be the new tab page, unless the URL is set to 'edge:/<br/><br/>ewtab'. If not configured users can choose whether the new tab page is their home page. This policy is available only on Windows instances that are joined to a Microsoft Active Directory domain or Windows 10 Pro or Enterprise instances enrolled for device management. | Disabled
+Microsoft Edge\Startup, home page and new tab page\Show Home button on toolbar | Shows the Home button on Microsoft Edge's toolbar. Enable this policy to always show the Home button. Disable it to never show the button. If you don't configure the policy, users can choose whether to show the home button. | Enabled
```

Comment:
Thanks for the feedback. I understand where you're coming from on these 3 settings (Home page button takes you to Office, new tab button takes you to Bing). I'll need to think more about whether this should be a default recommendation. The sites don't really matter so much as it'll be up to the customer to decide which default sites they use (LMS, Office, Google, etc). Where I need to think more is does it make sense to have Home take you one place and new tab take you to another. Is that confusing for students?

Reply:
This was more of an exercise than anything else. I forgot for a moment that these are default recommended settings.

As a grey beard PC user, I typically like to enable the Home button on personal machines (ironically not in any of the browsers of the machine I'm currently typing from), as I never considered it a waste of screen real-estate. Since I'm the one configuring machines for my district, I figured it would be a good practice to have the Home page for our district's landing page, and new tab for search. Now that we're discussing this in the context of students and not faculty, I will re-evaluate this approach for student shared PCs. For staff, I will certainly make these user changeable settings.

The main idea here is to differentiate the new tab URL from the Home page URL, so that anyone playing with these settings (and using the defaults for testing), will be able to spot the difference.